src/main/kotlin/Main.kt

```diff
@@ -1,16 +1,11 @@
-
 import org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory
```

Comment:
you can directly write all of this code in Application.kt and directly register resource instead of registering Application.kt here